sources/Log.cpp

```diff
@@ -0,0 +1,56 @@
+#include "Log.h"
+#include <QtDebug>
+#include <QFile>
+#include <QTextStream>
+#include <QUuid>
+#include "KVAbort.h"
+#include <QDateTime> 
```

Comment:
Better to group headers in some way. At least, separate Qt headers and the project headers in separate groups.
```
#include "Log.h"
#include "SomeHeader.h"

#include <QFile>
#include <QTextStream>
#include <QUuid>
```